buffer/src/main/java/io/netty5/buffer/api/bytebuffer/NioBuffer.java

```diff
@@ -1132,25 +1132,25 @@ protected void makeInaccessible() {
     }
 
     private void checkRead(int index, int size) {
-        if (index < 0 | woff < index + size) {
+        if (index < 0 || woff < index + size) {
```

Comment:
actually these are correct and faster then using a ´||´ please revert all these changes

Reply:
Hello, @normanmaurer .
Thanks for reviewing the codes.
I just reverted it.